test/e2e/node/security_context.go

```diff
@@ -214,10 +214,11 @@ var _ = SIGDescribe("Security Context", func() {
 		e2eoutput.TestContainerOutput(ctx, f, "seccomp runtime/default", pod, 0, []string{"2"}) // seccomp filtered
 	})
 
-	ginkgo.It("should support seccomp default which is unconfined [LinuxOnly]", func(ctx context.Context) {
+	// With the graduation of SeccompDefault to GA in v1.27 we now use `RuntimeDefault` instead of `Unconfined` in the default test environment.
+	ginkgo.It("should support seccomp default which is RuntimeDefault [LinuxOnly]", func(ctx context.Context) {
 		pod := scTestPod(false, false)
 		pod.Spec.Containers[0].Command = []string{"grep", "ecc", "/proc/self/status"}
-		e2eoutput.TestContainerOutput(ctx, f, "seccomp default unconfined", pod, 0, []string{"0"}) // seccomp disabled
+		e2eoutput.TestContainerOutput(ctx, f, "seccomp default unconfined", pod, 0, []string{"2"}) // seccomp filtered
```

Comment:
> we now use `RuntimeDefault` instead of `Unconfined`

Should `scenarioName` parameter be set to "seccomp default RuntimeDefault"?

Reply:
Exactly, fixed as suggested :+1: